Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -161,27 +161,27 @@
  ################
  # BATH'S BATHS #
  ################
- building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+ building_type_btwk_aquae_sulis_baths_01:0 "Baths of Aquae Sulis"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans occupied and developed a British shrine into a resort. The shrine (dedicated to the god #italic Sulis#!) had been built around a series of natural springs, later influencing the area's Latin name - #italic Aquae Sulis#!. These springs would later provide the foundations for the magnificent bathing complex Aquae Sulis is famed for."
```

Comment:
later influencing the area's Latin name -> giving the area's Latin name
magnificent bathing complex -> magnificent baths